tests/post/steps/test_monitoring.py

```diff
@@ -148,6 +153,98 @@ def _check_object_exists():
     utils.retry(_check_object_exists, times=20, wait=3)
 
 
+@given(parsers.parse("the '{ns_name}' namespace exists"))
+def namespace_exists(k8s_client, ns_name):
+    client = k8s_client.resources.get(api_version="v1", kind="Namespace")
+    try:
+        client.get(name=ns_name)
+    except ApiException as exc:
+        if not exc.status == 404:
+            raise AssertionError(f"Unable to create namespace: {exc}")
+        try:
+            client.create({"metadata": {"name": ns_name}})
+        except ApiException as err:
+            raise AssertionError(f"Unable to create namespace: {err}")
+
+
+@given(parsers.parse("we have a dashboard ready with a specific UID"))
+def dashboard_ready(context):
+    cm_uid = str(uuid.uuid4())
```

Comment:
Missing import for this module

Reply:
:man_facepalming: conflict resolution gone wrong